index.js

```diff
@@ -458,7 +539,7 @@
   def('compose',
       {},
       [$.Function, $.Function, a, c],
-      function(f, g, x) { return f(g(x)); });
+      compose3);
```

Comment:
We should make this change a few lines above:

```diff
-  //. See also [`pipe`](#pipe).
+  //. See also [`B`](#B) and [`pipe`](#pipe).
```

Let's include a "see also" link in the description of [`S.flip`][1] as well.


[1]: https://github.com/plaid/sanctuary#flip